### ISSUE TITLE
Inline mixins don't work in macros

### ISSUE THREAD

**Issue by ComLock** [enonic/xp#6600]
Seen in 6.15.2 using Text component. Perhaps this is related to the new HTMLArea.

**Comment by ComLock** [comment]
Macro
```xml
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<mixin>
    <display-name>Alert</display-name>
    <items>
        <input name="title" type="TextLine">
            <label>Title</label>
            <occurrences minimum="1" maximum="1"/>
        </input>

		<input name="text" type="TextLine">
			<label>Text</label>
			<occurrences minimum="1" maximum="1"/>
		</input>

        <input name="actionLink" type="TextLine">
			<label>actionLink</label>
			<occurrences minimum="0" maximum="1"/>
		</input>

        <input name="actionText" type="TextLine">
			<label>actionText</label>
			<occurrences minimum="0" maximum="1"/>
		</input>

        <input name="alertType" type="ComboBox">
            <label>alertType</label>
            <occurrences minimum="1" maximum="1"/>
            <config>
                <option value="information">information</option>
                <option value="warning">warning</option>
                <option value="error">error</option>
            </config>
            <default>information</default>
        </input>

        <input name="expandable" type="CheckBox">
          <label>expandable</label>
          <occurrences minimum="0" maximum="1"/>
        </input>
    </items>
</mixin>
```

**Comment by ComLock** [comment]
Macro
```xml
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<macro>
    <display-name>Alert</display-name>
    <description></description>
    <form>
        <inline mixin="alert"/>
    </form>
</macro>
```

**Comment by ComLock** [comment]
Chaning to this does not help.

```xml
<inline name=“alert”>
`
``

**Comment by alansemenov** [comment]
This doesn't work in 7.0 either (with ``<inline name="alert"/>``).

**Comment by alansemenov** [comment]
Waiting for decision in https://github.com/enonic/xp/issues/6634

**Comment by vbradnitski** [comment]
implemented in #6634
